index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>wommy.io - home</title>
+
+		<link rel='stylesheet' href='./style.css'>
+		<style>
+
+		</style>
+	</head>
+	<body id='home'>
+		<h1>wommy.io</h1>
+		<div id='main'>
+			<div>
+				<h2>created by me, tommy williams</h2>
+				<p>check out my resume <a href="./resume.html">here.</a></p>
+			</div>
+			<div>
+				<h3>portfolio</h3>
+				<ul>
+					<li><a href="./brent/index.html" target="_blank">the cool brent page</a></li>
+					<li><a href="./mcchris/index.html" target="_blank">the new mcchris pages</a></li>
```

Comment:
Why target blank here?

Reply:
opening them in a new tab cognitively separates them

as they arent part of the same website, but rather sub projects that fall under a certain umbrella

Reply:
https://css-tricks.com/use-target_blank/